app/helpers/course_summaries_helper.rb

```diff
@@ -0,0 +1,156 @@
+module CourseSummariesHelper
+  include SubmissionsHelper
+
+  # Get JSON data for the table
+  def get_table_json_data
```

Comment:
Assignment Branch Condition size for get_table_json_data is too high. [29.15/15]